ci-operator/step-registry/etcd-encryption/conf/etcd-encryption-conf-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: etcd-encryption-conf
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
+  commands: etcd-encryption-conf-commands.sh
+  resources:
+    requests:
+      cpu: 200m
+      memory: 100Mi
+  env:
+  - name: ZONES_COUNT
```

Comment:
Seem like the step does not need this env.

Reply:
Removed env